modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustClientCodegen.java

```diff
@@ -66,12 +67,13 @@ public class RustClientCodegen extends AbstractRustCodegen implements CodegenCon
     public static final String BEST_FIT_INT = "bestFitInt";
     public static final String AVOID_BOXED_MODELS = "avoidBoxedModels";
 
-    @Setter protected String packageName = "openapi";
+    protected String packageName = "openapi";
     @Setter protected String packageVersion = "1.0.0";
     protected String apiDocPath = "docs/";
     protected String modelDocPath = "docs/";
     protected String apiFolder = "src/apis";
     protected String modelFolder = "src/models";
+    private String externCrateName = "openapi";
```

Comment:
instead of adding another variable, what about simply create an function called `getCrateName()` and we can keep the @Setter above?

Reply:
Makes sense. Why add more state. I was just blindly copying from the rust server code gen.

Updated in the latest commit: https://github.com/OpenAPITools/openapi-generator/pull/19207/commits/5f8bda343de8361aaf763484961c5716833d1ce1